### PR TITLE
Update QD to QL 1

### PR DESCRIPTION
**Pull request by brawner**
Quality level 1 for rcpputils

Signed-off-by: Stephen Brawner <brawner@gmail.com>

### CODE REVIEW

File: QUALITY_DECLARATION.md [hunk 1]
```diff
@@ -106,24 +106,30 @@ This includes:
...
+Current coverage statistics can be viewed [here](https://ci.ros2.org/job/nightly_linux_coverage/lastCompletedBuild/cobertura/).
 A description of how coverage statistics are summarized from this page, can be found in the ["ROS 2 Onboarding Guide"](https://index.ros.org/doc/ros2/Contributing/ROS-2-On-boarding-Guide/#note-on-coverage-runs).
 
 ### Performance [4.iv]
 
-`rcpputils` does not currently have performance tests.
+`rcpputils` follows the recommendations for performance testing of C/C++ code in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#performance), and opts to do performance analysis on each release rather than each change.
+
+System level performance benchmarks that cover features of `rcpputils` can be found at:
+* [Benchmarks](http://build.ros2.org/view/Rci/job/Rci__benchmark_ubuntu_focal_amd64/lastCompletedBuild/)
```
Comment (chapulina) [inline QUALITY_DECLARATION.md:117]: Same comment as `rcutils`, consider linking to the benchmark tables.

File: QUALITY_DECLARATION.md [hunk 2]
```diff
@@ -106,24 +106,30 @@ This includes:
...
 
-Results of the nightly linter tests can be found [here](http://build.ros2.org/view/Epr/job/Epr__rcpputils__ubuntu_bionic_amd64/lastBuild/testReport/rcpputils/).
+Results of the nightly linter tests can be found [here](http://build.ros2.org/view/Rpr/job/Rpr__rcpputils__ubuntu_bionic_amd64/lastBuild/testReport/rcpputils/).
 
 ## Dependencies [5]
 
 ### Direct Runtime ROS Dependencies [5.i]/[5.ii]
 
-`rcpputils` has the following runtime ROS dependencies:
+`rcpputils` has the following runtime ROS dependencies, all of which are at **Quality Level 1**.
```
Comment (chapulina) [inline QUALITY_DECLARATION.md:132]: I'll just note that this depends on https://github.com/ros2/rcutils/pull/317

File: QUALITY_DECLARATION.md [hunk 3]
```diff
@@ -106,24 +106,30 @@ This includes:
...
+* [Performance](http://build.ros2.org/view/Rci/job/Rci__nightly-performance_ubuntu_focal_amd64/lastCompletedBuild/)
+
+Changes that introduce regressions in performance must be adequately justified in order to be accepted and merged.
 
 ### Linters and Static Analysis [4.v]
 
 `rcpputils` uses and passes all the standard linters and static analysis tools for a C++ package as described in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#linters).
 
-Results of the nightly linter tests can be found [here](http://build.ros2.org/view/Epr/job/Epr__rcpputils__ubuntu_bionic_amd64/lastBuild/testReport/rcpputils/).
+Results of the nightly linter tests can be found [here](http://build.ros2.org/view/Rpr/job/Rpr__rcpputils__ubuntu_bionic_amd64/lastBuild/testReport/rcpputils/).
```
Comment (chapulina) [inline QUALITY_DECLARATION.md:126]:
```suggestion
Results of the nightly linter tests can be found [here](http://build.ros2.org/view/Rpr/job/Rpr__rcpputils__ubuntu_focal_amd64/lastBuild/testReport/rcpputils/).
```

File: QUALITY_DECLARATION.md [hunk 4]
```diff
@@ -64,23 +64,23 @@ There is documentation for all of the features, and new features require documen
...
+`rcpputils` has embedded API documentation and it is hosted [here](http://docs.ros2.org/latest/api/rcpputils/index.html).
 
 ### License [3.iii]
 
 The license for `rcpputils` is Apache 2.0, and a summary is in each source file, the type is declared in the `package.xml` manifest file, and a full copy of the license is in the [LICENSE](./LICENSE) file.
 
 There is an automated test which runs a linter that ensures each file has a license statement.
 
-Most recent test results can be found [here](http://build.ros2.org/view/Epr/job/Epr__rcpputils__ubuntu_bionic_amd64/lastBuild/testReport/rcpputils/)
+Most recent test results can be found [here](http://build.ros2.org/view/Rpr/job/Rpr__rcpputils__ubuntu_bionic_amd64/lastBuild/testReport/rcpputils/)
```
Comment (chapulina) [inline QUALITY_DECLARATION.md:75]:
```suggestion
Most recent test results can be found [here](http://build.ros2.org/view/Rpr/job/Rpr__rcpputils__ubuntu_focal_amd64/lastBuild/testReport/rcpputils/)
```

File: QUALITY_DECLARATION.md [hunk 5]
```diff
@@ -64,23 +64,23 @@ There is documentation for all of the features, and new features require documen
...
 
 ### Copyright Statements [3.iv]
 
 The copyright holders each provide a statement of copyright in each source code file in `rcpputils`.
 
 There is an automated test which runs a linter that ensures each file has at least one copyright statement.
 
-The results of the test can be found [here](http://build.ros2.org/view/Epr/job/Epr__rcpputils__ubuntu_bionic_amd64/lastBuild/testReport/rcpputils/copyright/).
+The results of the test can be found [here](http://build.ros2.org/view/Rpr/job/Rpr__rcpputils__ubuntu_bionic_amd64/lastBuild/testReport/rcpputils/copyright/).
```
Comment (chapulina) [inline QUALITY_DECLARATION.md:83]:
```suggestion
The results of the test can be found [here](http://build.ros2.org/view/Rpr/job/Rpr__rcpputils__ubuntu_focal_amd64/lastBuild/testReport/rcpputils/copyright/).
```

File: QUALITY_DECLARATION.md (hunk 4 above)
Comment (brawner) [reply QUALITY_DECLARATION.md]: Done

File: QUALITY_DECLARATION.md (hunk 5 above)
Comment (brawner) [reply QUALITY_DECLARATION.md]: Fixed

File: QUALITY_DECLARATION.md (hunk 1 above)
Comment (brawner) [reply QUALITY_DECLARATION.md]: Replaced link.

File: QUALITY_DECLARATION.md (hunk 3 above)
Comment (brawner) [reply QUALITY_DECLARATION.md]: Done

File: QUALITY_DECLARATION.md (hunk 2 above)
Comment (brawner) [reply QUALITY_DECLARATION.md]: switched to singular

File: QUALITY_DECLARATION.md [hunk 6]
```diff
@@ -106,24 +106,30 @@ This includes:
...
 
-Results of the nightly linter tests can be found [here](http://build.ros2.org/view/Epr/job/Epr__rcpputils__ubuntu_bionic_amd64/lastBuild/testReport/rcpputils/).
+Results of the nightly linter tests can be found [here](http://build.ros2.org/view/Rpr/job/Rpr__rcpputils__ubuntu_focal_amd64/lastBuild/testReport/rcpputils/).
 
 ## Dependencies [5]
 
 ### Direct Runtime ROS Dependencies [5.i]/[5.ii]
 
-`rcpputils` has the following runtime ROS dependencies:
+`rcpputils` has the following runtime ROS dependencies, which is at **Quality Level 1**.
```
Comment (chapulina) [inline QUALITY_DECLARATION.md:132]: Nit:

```suggestion
`rcpputils` has the following runtime ROS dependency, which is at **Quality Level 1**.
```

Also consider adding a link to `rcutils`' QD

Comment (brawner) [reply QUALITY_DECLARATION.md]: Whoops, fixed

**Review by chapulina — APPROVED** [review]

**Pull request merged**